docs/docs/getting-started/installation/gcp.md

```diff
@@ -0,0 +1,202 @@
+| [**VPC**](https://cloud.google.com/vpc/docs/vpc)                                       | 1 VPC, 36 subnets with allowed Public IPs, 1 additional route table, 1 Internet gateway, DNS enabled | [ VPC pricing calculation](https://cloud.google.com/vpc/network-pricing#lb)                                      | VPC which will contain all the created compute and network resources                                                                                       | Yes            |
+| [**GKE**](https://cloud.google.com/kubernetes-engine/docs)                             | 1 GKE cluster                                                                                        | [ GKE pricing calculation](https://cloud.google.com/products/calculator#id=e178295f-5232-4319-8a16-708052357202) | Kubernetes cluster to store all the Airy Core resources                                                                                                    | No             |
+| [**GCE**](https://cloud.google.com/kubernetes-engine/docs/concepts/node-images#cos)    | 6 Google Compute Engine Instances                                                                    | [GCE pricing calculation](https://cloud.google.com/products/calculator#id=0394acda-ce3a-4769-a453-9e2ea6ca4291)  | The instances are a part of the `Node Pool` attached to the GKE cluster. The default instance type is: `n1-standard-2`, OS type: `Container-Optimized OS`. | Yes            |
+| [**CLB**](https://cloud.google.com/load-balancing/docs/network/networklb-target-pools) | 1 Cloud Load Balancer                                                                                | [CLB pricing calculation](https://cloud.google.com/products/calculator#id=da086850-b923-475e-a4bc-9edf3d73c9ed)  | Network (target pool-based)Load Balancer created by the ingress controller Kubernetes service                                                              | No             |
+
+[^1]: Options which can be overwritten with flags to the `airy create` command.
+
+## Install
+
+To create the cluster you must setup your local GCP environment, by [configuring
+your local GCP
+profile](https://cloud.google.com/sdk/docs/install-sdk)
+for the GCP account where all the resources will be created.
+
+Download and install the [Airy CLI](cli/introduction.md).
+
+:::warning
+
+If you want to use Airy Core with auto-generated HTTPS certificates, refer to the [Let's Encrypt section](/getting-started/installation/gcp#https-using-lets-encrypt) for customizing your `airy.yaml` file before proceeding.
```

Comment:
The `Let's Encrypt section` link doesn't work.

Reply:
👍 *fixed*